### ISSUE TITLE
support parsing datetime format for Synology custom log synocmsclient.log

### ISSUE THREAD

**Issue by jtmoon79** [jtmoon79/super-speedy-syslog-searcher#47]
## Problem

Synology DiskStation OS has [an unsupported syslog datetime format in file `synocmsclient.log`](https://github.com/jtmoon79/super-speedy-syslog-searcher/blob/0.0.34/logs/synology-DS6/synocmsclient.log)

```text
Mon Dec  5 20:31:50 PST 2016
synogpoclientd: not join cms, need not start
```

## Solution

Support the ad-hoc message datetime format.  Add test cases in `datetime.rs` and comment where they originate.

**Comment by jtmoon79** [comment]
I thought there was already support for this ad-hoc datetime format.  This needs some investigation.

**Comment by jtmoon79** [comment]
Fixed in 1e58094eafae95c9c09b35c63aa000a0edfd5845
